### PR TITLE
[ACTION] Zoom new action list webinar participants report plus refactor

### PR DESCRIPTION
**Pull request by jcortes**

### CODE REVIEW

File: components/zoom/sources/meeting-ended/meeting-ended.mjs [hunk 1]
```diff
@@ -1,36 +1,31 @@
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
   ...common,
   key: "zoom-meeting-ended",
   name: "Meeting Ended (Instant)",
   description: "Emit new event each time a meeting ends where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/meeting-ended/meeting-ended.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/meeting-started/meeting-started.mjs [hunk 2]
```diff
@@ -1,28 +1,21 @@
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
   ...common,
   key: "zoom-meeting-started",
   name: "Meeting Started (Instant)",
   description: "Emit new event each time a meeting starts where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/meeting-started/meeting-started.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/meeting-updated/meeting-updated.mjs [hunk 3]
```diff
@@ -1,27 +1,20 @@
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
   ...common,
   key: "zoom-meeting-updated",
   name: "Meeting Updated (Instant)",
   description: "Emit new event each time a meeting is updated where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/meeting-updated/meeting-updated.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/phone-event/phone-event.mjs [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
+import common from "../common/common.mjs";
 import constants from "../common/constants.mjs";
 
 export default {
+  ...common,
   key: "zoom-phone-event",
   name: "Zoom Phone Events (Instant)",
-  description: "Listen for any Zoom Phone events tied to your Zoom user or resources you own",
-  version: "0.0.2",
+  description: "Emit new Zoom Phone event tied to your Zoom user or resources you own",
```
Comment (alysonturing) [inline components/zoom/sources/phone-event/phone-event.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/recording-completed/recording-completed.mjs [hunk 5]
```diff
@@ -1,48 +1,47 @@
-import zoom from "../../zoom.app.mjs";
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
+  ...common,
   key: "zoom-recording-completed",
   name: "Recording Completed (Instant)",
   description: "Emit new event each time a new recording completes for a meeting or webinar where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/recording-completed/recording-completed.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/webinar-created/webinar-created.mjs [hunk 6]
```diff
@@ -1,28 +1,20 @@
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
   ...common,
   key: "zoom-webinar-created",
   name: "Webinar Created (Instant)",
   description: "Emit new event each time a webinar is created where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/webinar-created/webinar-created.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/webinar-deleted/webinar-deleted.mjs [hunk 7]
```diff
@@ -1,25 +1,22 @@
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
   ...common,
   key: "zoom-webinar-deleted",
-  name: "Webinar Deleted",
+  name: "Webinar Deleted (Instant)",
   description: "Emit new event each time a webinar is deleted where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/webinar-deleted/webinar-deleted.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/webinar-ended/webinar-ended.mjs [hunk 8]
```diff
@@ -1,30 +1,23 @@
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
   ...common,
   key: "zoom-webinar-ended",
   name: "Webinar Ended (Instant)",
   description: "Emit new event each time a webinar ends where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/webinar-ended/webinar-ended.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/webinar-started/webinar-started.mjs [hunk 9]
```diff
@@ -1,27 +1,20 @@
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
   ...common,
   key: "zoom-webinar-started",
   name: "Webinar Started (Instant)",
   description: "Emit new event each time a webinar starts where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/webinar-started/webinar-started.mjs:8]: is it possible to add the link to the docs?

File: components/zoom/sources/webinar-updated/webinar-updated.mjs [hunk 10]
```diff
@@ -1,27 +1,20 @@
 import common from "../common/common.mjs";
+import constants from "../common/constants.mjs";
 
 export default {
   ...common,
   key: "zoom-webinar-updated",
   name: "Webinar Updated (Instant)",
   description: "Emit new event each time a webinar is updated where you're the host",
```
Comment (alysonturing) [inline components/zoom/sources/webinar-updated/webinar-updated.mjs:8]: is it possible to add the link to the docs?

**Review by alysonturing — CHANGES_REQUESTED** [review]
Thanks for your contributions, I just added some minor changes to improve usability and ensure our pattern

**Review by alysonturing — DISMISSED** [review]
LGTM!

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Zoom_5597_0948.pdf](https://github.com/PipedreamHQ/pipedream/files/10904926/Zoom_5597_0948.pdf)

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Zoom_5597_7001.pdf](https://github.com/PipedreamHQ/pipedream/files/10926363/Zoom_5597_7001.pdf)

**Review by GTFalcao — APPROVED** [review]

**Pull request merged**